src/components/LoginPage/LoginPage.jsx

```diff
@@ -1,18 +1,92 @@
-import React from 'react';
-import { Button } from 'react-bootstrap';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Is there a reason you have to import bootstrap like this? I thought I installed the packages using npm. Maybe you have to add that to `index.html`?

Reply:
I thought I had installed bootstrap with npm i already but the styling wasnt showing up when i used any bootstrap components so I added this to include the styling sheet and it worked. Ill try the npm -i bootstrap again

Reply:
It seems like I have to have the stylesheet included. I tried running the npm install commands but once I removed the import statement there was no bootstrap styling. Not sure why because the initial LoginPage you created didn't seem to have that issue.

Reply:
Hmm that's weird. I guess it's fine for now we can figure out how to get rid of the import statement later.

Reply:
ok sounds good